tests/jballerina-unit-test/src/test/resources/test-src/expressions/literals/identifierliteral/identifier-literal-wrong-character-negative.bal

```diff
@@ -1,6 +1,7 @@
 
 function getGlobalVarWithIL() returns string {
     string 'global\ var" = "dfs";
+    int \
```

Comment:
Shall we add this as a  parser test as well. since the issue is related to the tokenizing?

Reply:
Added